legacy/plugin.ts

```diff
@@ -89,7 +89,8 @@ export interface PluginMetadata {
 
   // Per-plugin custom metadata
   meta?: {
-    allSubProjectNames?: string[];
+    allSubProjectNames?: string[],
+    gradleProjectName?: Array<Map<string, string>>;
```

Comment:
Looks like this can just be `string[]`? What is the data example this will hold?

Reply:
format will be:
```
{
  "packageName": "oldPackageName",
  "newPackageName:" "newPackageName" 
}
```

The same format will be for multi projects as well, part of `ScannedProject`